_articles/scripting/Typescript/typescript-modifier.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Modifiers in Typescript
+author: Shush
+steamId: 76561197994333648
+date: 07.03.2021
+---
+
+Modifiers are an extremely important part of almost any Dota addons. They allow you to modify certain properties of your hero, deal damage to it over time, or apply various effects on it. Like abilities, we'll also create them in Typescript.
```

Comment:
I think using 'custom game' instead of 'addon' is nicer.

Reply:
Replaced with custom game.